tools/apidiff/cmd/root.go

```diff
@@ -34,6 +35,7 @@ individual packages or a set of packages under a specified directory.`,
 }
 
 func init() {
+	rootCmd.PersistentFlags().BoolVarP(&copyRepoFlag, "copyrepo", "c", false, "copy the repo instead of cloning it")
```

Comment:
>copyRepoFlag [](start = 37, length = 12)

when would this be useful ? also is the breaking change tool doing any distructive behavior on the code itself ?

Reply:
It's useful for CI (see the change description where I go into more detail).